### DISCUSSION TITLE
Current Cash Register Details button on POS "Feature Suggestion"

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#569] [Ideas]
Any cashier while closing his cash he need to make an excel report to compare his sales with the current balance in cash drawer and the only solution is to give him access to the cash flow report or payment report which should not be done because he can see whole month or year sales. All POS is making Cash Register Details button on POS which show all details for the current author and current openeing cash register

![register details Nexo POS feature suggestion](https://user-images.githubusercontent.com/87772530/153730650-60c39136-18fd-464a-abac-782c27932221.png)

**Comment by Blair2004** [comment]
approved

**Comment by magedfathi** [comment]
can we have print option after displaying this cash register details
